### PR TITLE
docs: tutorial summary page

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

**Comment by nickcernera** [comment]
once this is merged, my PR (e.g. new homepage) should be updated so that the "Tutorials" card links to the new Tutorials page

File: docs/pages/tutorials.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# MUD tutorials
+
+These tutorials teach you how to do various things with MUD.
+
+## [Modifying the getting started example](tutorials/minimal)
```
Comment (holic) [inline docs/pages/tutorials.mdx:5]: These linked headings look kinda bad, wonder if @nickcernera has ideas for spicing them up?
https://mud-docs-1dqbb3x0k-latticexyz.vercel.app/tutorials

File: docs/pages/tutorials.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- [Add a system](tutorials/minimal/add_system):
+  In this tutorial you add a system to decrement the counter and update the application to use it.
+
+## [Walkthrough](tutorials/walkthrough)
+
+These tutorials are deep dives into various sections of code.
+
+- [The onchain components of getting started](tutorials/walkthrough/onchain):
+  In this tutorial you learn how to understand the onchain components of the minimal template.
```
Comment (holic) [inline docs/pages/tutorials.mdx:18]: What about Emojimon?

File: docs/pages/tutorials.mdx (hunk 1 above)
Comment (nickcernera) [reply docs/pages/tutorials.mdx:5]: I slotted yesterday for MUD community/docs stuff but need to pivot back to Sky Strife the rest of the week — can you figure something out in the meantime?

Also achal will be in the docs prettifying everything soon enough!

File: docs/pages/tutorials.mdx (hunk 2 above)
Comment (qbzzt) [reply docs/pages/tutorials.mdx:18]: Added.

File: docs/pages/tutorials.mdx (hunk 1 above)
Comment (qbzzt) [reply docs/pages/tutorials.mdx]: Links moved.

**Review by alvrs — APPROVED** [review]

**Review by alvrs — APPROVED** [review]

**Pull request merged**